build.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+fn main() -> std::io::Result<()> {
+    panic!(
+        r#"This crate has been replaced by the jj-cli crate.
+
+Please upgrade to jj-cli version 0.8+, or specify jujutsu of version "=0.7.0" if you need to compile an obsolete version."#
+    );
```

Comment:
Turns out you can't publish a crate that fails to build, so I'll have to remove the build.rs files.